dojo_theme/templates/dojo_admin.html

```diff
@@ -28,6 +28,16 @@ <h2>{{ dojo.name }}</h2>
       </a>
       </figure>
     </div>
+    {% if is_admin() %}
+    <div class="col-lg-auto m-3">
+      <figure>
+        <a class="text-decoration-none" href="{{ url_for('pwncollege_dojos.delete_dojo', dojo=dojo.reference_id,update_code=dojo.update_code) }}" target="_blank">
```

Comment:
As mentioned in another comment, let's make this a POST form similar to `dojo-promote-dojo`.

Reply:
okay，now delete function is POST